lib/clib/CCorres_Rewrite.thy

```diff
@@ -17,7 +17,7 @@ lemma ccorres_com_eq_hom:
            elim: ccorres_semantic_equivD2)
 
 method ccorres_rewrite declares C_simp C_simp_pre C_simp_simps C_simp_throws
-  = simpl_rewrite hom: ccorres_com_eq_hom
+  = (simpl_rewrite hom: ccorres_com_eq_hom, no_name_eta)
```

Comment:
Interesting, do you think ccorres_rewrite can get into a situation that requires no_name_eta?

Reply:
I unfortunately don't have the example anymore, but that was actually what first sparked all of this work. A `ccorres_rewrite` left the goal eta-expanded and a following step then rewrote `dc` when I didn't want it to.

Reply:
Unexpected! In this case even more of a thumbs up.